### PR TITLE
lfortran 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

- Website: https://lfortran.org/
- GitHub: https://github.com/lfortran/lfortran
- Requires LLVM 9 and Python 3.7
- Currently unable to use `lfort` to compile fortran code directly to a binary (i.e. `lfort -o hello hello.f90` doesn't work and I've tried fixing it without much success)
- I probably should add a comment that says antlr4-python3-runtime must be pinned to version 4.7

### CODE REVIEW

File: Formula/lfortran.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+  desc "Modern interactive LLVM-based Fortran compiler"
+  homepage "https://lfortran.org/"
+  url "https://lfortran.github.io/tarballs/release/lfortran-0.2.0.tar.gz"
+  sha256 "729b38269b1c933f3b2dc9253e1fab7e7aec4206d518e9cf007b171da5af53c3"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@9"
+  depends_on "python"
```
Comment (chenrui333) [inline Formula/lfortran.rb:12]: maybe `python@3.8`?

Comment (SeekingMeaning) [reply Formula/lfortran.rb:12]: Nope, LFortran links to Python 3.7 methods and gives compile errors with Python 3.8

Comment (SMillerDev) [reply Formula/lfortran.rb:12]: This is pointing to python@3.8

File: Formula/lfortran.rb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  include Language::Python::Virtualenv
+
+  desc "Modern interactive LLVM-based Fortran compiler"
+  homepage "https://lfortran.org/"
+  url "https://lfortran.github.io/tarballs/release/lfortran-0.2.0.tar.gz"
+  sha256 "729b38269b1c933f3b2dc9253e1fab7e7aec4206d518e9cf007b171da5af53c3"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@9"
```
Comment (SMillerDev) [inline Formula/lfortran.rb:11]: We don't accept new formula that depend on outdated versions.

**Pull request closed without merging**